.github/workflows/build-frontend.yml

```diff
@@ -31,6 +31,9 @@ jobs:
           yarn pretty-quick --check --verbose --branch ${{ inputs.target_branch }}
           yarn test
 
+      - name: Create version file
+        run: jq -r '.version' package.json > frontend/build/version.txt
```

Comment:
I thought we got rid of the version.txt file? I know we don't need it anymore. I suspect this can go away.

Reply:
I am referring to this: https://github.com/kiali/kiali/pull/4903/files

Reply:
Thanks, seems that yes, it can go away.